src/app/features/tech-record/components/tech-record-search-tyres/tech-record-search-tyres.component.html

```diff
@@ -91,8 +91,12 @@ <h3 class="govuk-heading-m">{{ searchResults.length }} search result{{ searchRes
         <td class="govuk-table__cell">{{ tyre.plyRating }}</td>
         <td class="govuk-table__cell">{{ tyre.loadIndexSingleLoad }}</td>
         <td class="govuk-table__cell">{{ tyre.loadIndexTwinLoad }}</td>
-        <td class="govuk-table__cell">{{ tyre.axleLoadSingle | tyreAxleLoad : tyre.loadIndexSingleLoad : 2 | defaultNullOrEmpty }}</td>
-        <td class="govuk-table__cell">{{ tyre.axleLoadDouble | tyreAxleLoad : tyre.loadIndexTwinLoad : 4 | defaultNullOrEmpty }}</td>
+        <td class="govuk-table__cell">
+          {{ tyre.axleLoadSingle | tyreAxleLoad : tyre.loadIndexSingleLoad : 2 : (loadIndex$ | async) | defaultNullOrEmpty }}
+        </td>
+        <td class="govuk-table__cell">
+          {{ tyre.axleLoadDouble | tyreAxleLoad : tyre.loadIndexTwinLoad : 4 : (loadIndex$ | async) | defaultNullOrEmpty }}
+        </td>
```

Comment:
Wondering if the following reads a little more clearly... what do you think?

```suggestion
        <td class="govuk-table__cell">
          {{ tyre.axleLoadSingle | tyreAxleLoad : tyre.loadIndexSingle : 2 : (loadIndex$ | async) | defaultNullOrEmpty }}
        </td>
        <td class="govuk-table__cell">
          {{ tyre.axleLoadDouble | tyreAxleLoad : tyre.loadIndexDouble : 4 : (loadIndex$ | async) | defaultNullOrEmpty }}
        </td>
```

Reply:
Can't change that. The name comes from the tyre reference data.